Providers/SetProvider.tscn

```diff
@@ -9,11 +9,15 @@ signal selected_day_night()
 var _drops_tables := [] # TODO: Maybe use a dictionary instead
 var _enemy_sets := []
 var _gathering_spots := []
+var store_ToD = 0
 
+func set_ToD():
```

Comment:
This function won't be necessary. And if someone wanted to do it anyways (which they shouldn't), one could still call SetProvider.select_day_night(selected_day_night)

Reply:
It's not the function, selected_day_night is the new name of the variable that was previously called store_ToD